database/migrations/2021_03_17_110938_update_default_avatar_users_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateDefaultAvatarUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('avatar')->default(null)->change();
```

Comment:
sorry my bad. Please change this to 
```suggestion
            $table->string('avatar')->nullable()->change();
```

di diay ta ka set og default null.

after changing the code please run `php artisan migrate:refresh`